app/views/unattended/provisioning_templates/registration/linux_registration_default.erb

```diff
@@ -20,6 +20,17 @@ set -e
 <%= snippet 'puppet_setup' %>
 <% end -%>
 
+<%= snippet 'remote_execution_ssh_keys' %>
+
+<% if @host.operatingsystem.name == 'RedHat' && host_param_true?('host_registration_insights') -%>
+echo '#'
+echo '# Installing Insights client'
+echo '#'
+
+yum install -y insights-client
+insights-client --register
+<% end -%>
```

Comment:
Does it make sense to create this as a separate snippet?

Reply:
Not sure if it is worthy. It's just for RHEL and it's basically just two commands ...

Reply:
I think it would make sense since we will probably want to reuse this in a kickstart template too. Keeping for another PR though, this template will change in next steps, so we'll extract the final version. @stejskalleos could you please create redmine issue for this? The subject is to extract insights initialization to partial and add it to kickstart templates.

Reply:
That was indeed my thought as well. We should try to keep parity between kickstarts and registration as much as possible and snippets can be very useful in achieving that (as you saw with REX already).